actions/app.go

```diff
@@ -29,32 +25,28 @@ func App() http.Handler {
 	})
 	log.Println("Environment:", ENV)
 	log.Println("Assets:", assetsPath())
-	a.Use(setTemplate())
+	a.Use(setLayout())
 	a.Use(middleware.PopTransaction(models.DB))
 	a.ServeFiles("/assets", assetsPath())
-	a.GET("/", CountdownHandler)
 	a.GET("/home", HomeHandler)
+	a.GET("/", CountdownHandler)
 	adm := a.Group("/admin")
 	adm.GET("/index", AdminHandler)
 
 	return a
 }
 
-func setTemplate() buffalo.MiddlewareFunc {
+func setLayout() buffalo.MiddlewareFunc {
```

Comment:
@bketelsen you have a  concurrency issue here. it's possible for two requests to come in at the same time and override the other one's preferred layout. you should use a separate renderer for the admin stuff.